Extension/src/main.ts

```diff
@@ -427,7 +430,7 @@ function checkDistro(channel: vscode.OutputChannel, platformInfo: PlatformInform
     }
 }
 
-function rewriteManifest(installBlob: InstallBlob): Promise<void> {
+function rewriteManifest(installBlob: InstallBlob): void {
```

Comment:
Did you mean to make this change? I don't see the corresponding change on the caller.

Reply:
Since I'm not returning anything now, this change is needed to compile. The caller still waits for the function to complete.